mypy/reachability.py

```diff
@@ -150,8 +150,8 @@ def consider_sys_version_info(expr: Expression, pyversion: Tuple[int, ...]) -> i
     return TRUTH_VALUE_UNKNOWN
 
 
-def consider_sys_platform(expr: Expression, platform: str) -> int:
-    """Consider whether expr is a comparison involving sys.platform.
+def consider_sys_platform(expr: Expression, platform: str, platform_system: str) -> int:
+    """Consider whether expr is a comparison involving sys.platform
```

Comment:
Please update the docstring to add that this also considers `platform.system` (see below).